src/gwt/src/org/rstudio/core/client/theme/res/themeStyles.css

```diff
@@ -401,6 +405,10 @@ pre {
    width: 7px;
 }
 
+.rstudio-themes-flat.rstudio-themes-dark-menus .gwt-MenuBar .subMenuIcon img {
+   filter: invert(100%);
```

Comment:
Is this supported on QtWebKit? Does it invert any toolbar icon colors? (seems like that could get weird!)